clang/test/SemaHLSL/resource_binding_attr_error_mismatch.hlsl

```diff
@@ -0,0 +1,65 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.3-library -x hlsl -o - -fsyntax-only %s -verify
+
+
+// expected-error@+1 {{invalid register name prefix 'b' for register type 'RWBuffer' (expected 'u')}}
+RWBuffer<int> a : register(b2, space1);

```

Comment:
Could we add a test for RWBuffer<int> b: register();/RWBuffer<int> b: register(""); to make sure empty string works?